tests/flag_util_test.py

```diff
@@ -100,3 +100,84 @@ def testSerialize(self):
 
     self.assertEqual(ser.Serialize(il),
                      '1,2-5,9')
+
+
+class TestObjectSize(unittest.TestCase):
+  def testStrSmall(self):
+    size = flag_util.ObjectSize(bytes=3)
+
+    self.assertEquals(str(size), '3B')
+
+  def testStrBase10(self):
+    size = flag_util.ObjectSize(bytes=3 * (1000 ** 2))
```

Comment:
Consider allowing the constructor to accept a string - it seems like it could be a common use case.